File: euler_p1.js
//Multiples of 3 and 5
//Problem # 1
//Build using JSC

function find_sum() { 
	sum = 0;
	for (i = 0; i < 1000; i++) {
		if (i % 3 == 0 || i % 5 == 0) {
			sum += i
		}	
	}
	return sum;
}

debug(find_sum());

File: euler_p1.py
#!/usr/bin/env python

#Multiples of 3 and 5
#Problem # 1

import time
import pdb

t1 = time.time()

sum = 0

for i in range(0,1000):
    if i % 3 == 0 or i % 5 == 0:
        sum += i	

print(sum)

t2 = time.time() - t1
print("runner time: %s" % t2)
